libvuln/jsonblob/jsonblob.go

```diff
@@ -289,12 +435,26 @@ func (s *Store) UpdateEnrichments(ctx context.Context, kind string, fp driver.Fi
 	return ref, nil
 }
 
-// RecordUpdaterStatus is unimplemented
+// RecordUpdaterStatus is unimplemented.
 func (s *Store) RecordUpdaterStatus(ctx context.Context, updaterName string, updateTime time.Time, fingerprint driver.Fingerprint, updaterError error) error {
 	return nil
 }
 
-// RecordUpdaterSetStatus is unimplemented
+// RecordUpdaterSetStatus is unimplemented.
 func (s *Store) RecordUpdaterSetStatus(ctx context.Context, updaterSet string, updateTime time.Time) error {
 	return nil
 }
+
+var bufPool sync.Pool
+
+func getBuf() []byte {
+	const sz = 1 << 20 // 1MiB
```

Comment:
This is the max token size for `bufio.Buffer`, right? Seems reasonable. I'd be surprised if a single vuln entry were over 1MB in size